webpack/assets/javascripts/react_app/components/Layout/LayoutConstants.js

```diff
@@ -3,3 +3,6 @@ export const LAYOUT_HIDE_LOADING = 'LAYOUT_HIDE_LOADING';
 export const LAYOUT_CHANGE_LOCATION = 'LAYOUT_CHANGE_LOCATION';
 export const LAYOUT_CHANGE_ORG = 'LAYOUT_CHANGE_ORG';
 export const LAYOUT_UPDATE_ITEMS = 'LAYOUT_UPDATE_ITEMS';
+export const LAYOUT_LOGOUT = 'LAYOUT_LOGOUT';
+
+export const LAYOUT_LOGOUT_URL = 'users/logout';
```

Comment:
this doesnt include the relative path, you should consider using foremanUrl function.